ros/license.md

```diff
@@ -1,3 +1,3 @@
-The core of ROS is licensed under the standard three-clause BSD license. This is a very permissive open license that allows for reuse in commercial and closed source products. You can find more about the BSD license from the Opensource.org [BSD 3-Clause](http://opensource.org/licenses/BSD-3-Clause) page and Wikipedia's [BSD Licenses](http://en.wikipedia.org/wiki/BSD_licenses) entry.
+The core of ROS is licensed under the Apache License 2.0. This is a very permissive open license that allows for modification, distribution, commercial use, patent use, and private use in open and closed source products. You can find more about the license from the [Apache License](https://www.apache.org/licenses) website or Opensource.org [Apache License 2.0](https://opensource.org/licenses/Apache-2.0) page.
```

Comment:
Not sure how to deal with this but the 2 project (ROS and ROS 2) have different license for their core and this change would apply only to ROS 2

Reply:
We can simply follow with other library images and link to external resources for finding licences of included packages: https://github.com/docker-library/docs/pull/1381/commits/23bf362b2369c36acefb18fa7d85559fe1cf72aa